plugins/repository-s3/src/main/java/org/elasticsearch/cloud/aws/AwsS3Service.java

```diff
@@ -76,6 +77,11 @@
      */
     Setting<String> REGION_SETTING =
         new Setting<>("cloud.aws.region", "", s -> s.toLowerCase(Locale.ROOT), Property.NodeScope, Property.Shared);
+    /**
+     * cloud.aws.read_timeout: Socket read timeout. Shared with discovery-ec2 plugin
+     */
+    Setting<TimeValue> READ_TIMEOUT = Setting.timeSetting("cloud.aws.read_timeout", TimeValue.timeValueSeconds(50),
```

Comment:
@dadoonet What can i use for default value? Because it's not possible to create timeSetting without default value. Maybe `TimeValue.MINUS_ONE`